apps/wallet-mobile/src/features/AddWallet/common/StepperProgress/StepperProgress.tsx

```diff
@@ -0,0 +1,118 @@
+import {useTheme} from '@yoroi/theme'
+import React from 'react'
+import {StyleSheet} from 'react-native'
+import Animated, {Layout} from 'react-native-reanimated'
+
+import CheckIcon from '../../illustrations/CheckIcon'
+
+type StepProps = {
+  currentStep: number
+  currentStepTitle?: string
+  todoStep: boolean
+  pastStep: boolean
+  displayStepNumber?: boolean
+}
+const Step = ({currentStep, currentStepTitle, displayStepNumber, todoStep, pastStep}: StepProps) => {
+  const {styles} = useStyles()
+  return (
+    <>
```

Comment:
just confirming that this view is part of the previous view, so the fragment usage here is intentional ?